webapp/client/src/lib/propTypes.js

```diff
@@ -9,3 +9,14 @@ export const fieldPropType = PropTypes.exact({
   value: PropTypes.any,
   errors: PropTypes.arrayOf(PropTypes.string),
 });
+
+export const selectionFieldPropType = PropTypes.exact({
```

Comment:
If this were called `dropDownFieldPropType`, the link to the `FormFieldDropDown` would be more obvious to me.